templates/web_macros.bicep

```diff
@@ -16,6 +16,10 @@
   name: 'POSTGRES_PASSWORD'
   secretRef: 'dbserver-password'
 }
+{
```

Comment:
@kjaymiller Is this file used anywhere? I searched and didnt find an obvious reference to it.

Reply:
I don't think it is but let's remove it in a separate PR in case we need to revert.